RSDK-695: only call supported movement sensor methods in RC

stevebriskin opened the pull request:
Approach: get properties first then selectively call other methods. there's a bit of a race condition on the first run as GetProperties is async, but it resolves itself in subsequent refreshes.

Manually tested with the fake config. Manually modified the fake movement sensor's Properties to expose different methods.

micheal-parks commented:
Work here conflicts with refactors [made in this PR](https://github.com/viamrobotics/rdk/pull/1373), so I've incorporated your changes. (You can see them in the new [`movement-sensor.vue`](web/frontend/src/components/movement-sensor.vue) component). I've also removed the race condition: properties are now always fetched before other sensor data.

stevebriskin commented:
> Work here conflicts with refactors [made in this PR](https://github.com/viamrobotics/rdk/pull/1373), so I've incorporated your changes. (You can see them in the new [`movement-sensor.vue`](https://github.com/viamrobotics/rdk/blob/957ced89cbc1c3eaf1bb3466ac65337eaa52aafe/web/frontend/src/components/movement-sensor.vue) component). I've also removed the race condition: properties are now always fetched before other sensor data.

Thank you! opened https://github.com/viamrobotics/rdk/pull/1398 as a follow up.
